### PR TITLE
Added Deb's PS and minor but important edits to notes (which should be published)

### PR DESCRIPTION
**Pull request by shomurali**

### CODE REVIEW

**Comment by andrewpbray** [comment]
thank you for the very informative PR name!

File: 4-generalization/18-chance-intro/notes.qmd [hunk 1]
```diff
@@ -94,7 +92,7 @@ data.frame(box) %>%
 ```
 
 
-Let's make a table of all the tickets and their chances. This is called a *distribution table*. 
+Let's make a table of all the tickets and their chances. This is called a *probability distribution table*.
```
Comment (andrewpbray) [inline 4-generalization/18-chance-intro/notes.qmd:95]: these are all great fixes!

**Comment by andrewpbray** [comment]
Since we told students that part of the activity from Weds would be on the problem set could we either:

1. Add to this PS: Ask that they answer one of the questions for 1 of the 8 scenarios in the slides
2. Provide an alternative: Instead of doing the one on the PS they could alternatively do one from the activity.

File: 4-generalization/18-chance-intro/notes.qmd (hunk 1 above)
Comment (shomurali) [reply 4-generalization/18-chance-intro/notes.qmd:95]: Yeah, these are mostly Deb's edits :)

**Comment by andrewpbray** [comment]
thanks!

**Pull request merged**